### PR TITLE
Complete the html strcture

### PR DESCRIPTION
**Pull request by misikir21**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,26 +1,36 @@
 {
-  "name": "leader-board-api",
+  "name": "to-do-list",
```
Comment (souad988) [inline package.json]: Good job by correctly configuring web pack 👍🏼  but why did you change the project name here?!!

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,3 @@
+html {
+  font-size: 10px;
+}
```
Comment (souad988) [inline src/style.css:3]: Please make sure to create the same wireframe as mentioned in this projects requirements it is very basic  but you are asked to folow it 


Capture d’écran du 2023-08-02 13-38-58
Capture d’écran du 2023-08-02 13-38-44

**Review by souad988 — CHANGES_REQUESTED** [review]
Hi @misikir21 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

242722065-db428b0c-8dfc-49c4-a156-42f227e32868


## Highlights
Configure webpack correctly
Professional README

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by abdulhamiid — APPROVED** [review]
# :green_circle:  Approved! :partying_face:
Hi @misikir21 👋, 
image
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 🌟

- ✔️ Setup project
- ✔️ Linters checks are passing

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

# OPTIONAL
- [ ] Please, let's always document our work in a professional way 🙏

- [ ] Writing a meaningful title for our `pull request` and also add a `short summary` of changes introduced in the pull request 😊

- [ ] Descriptive `commit messages` - each commit should be small enough to describe it in one sentence. The commit message should be written in the imperative tense and capitalized. [Read more about how to write descriptive commit messages](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8). 👍

- [ ] Professional repo - basic rules [Learn more](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)

Cheers, and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**